website/blog/2022/04/29/interview-juzhiyuan-apache-member.md

```diff
@@ -0,0 +1,340 @@
+---
+title: "Interview with JU Zhiyuan: Becoming One of the 918 ASF Members, I Felt Thrilled and Proud"
```

Comment:
What is 918 ASF Member? Maybe 90s?

Reply:
Looks like there are 900+ members, ref: https://www.apache.org/foundation/members.html

Reply:
I wrote the English title from a different perspective. As you know, there are a total of 918 ASF members globally as of today. Data speaks and this piece of data speaks louder because it is more powerful and convincing than ''the 90s''.